luigi/execution_summary.py

```diff
@@ -351,6 +353,8 @@ def _summary_format(set_tasks, worker):
     for status in _ORDERED_STATUSES:
         if status not in comments:
             continue
+        if status == 'ever_failed' and (set_tasks["ever_failed"] == set_tasks["failed"]):
```

Comment:
Hmm.
I think the latter condition is no longer necessary.
Because we don't show about `ever failed` in execution summary,  we should just skip in `status == 'ever_failed'` like

```
         if status == 'ever_failed':
             continue
```

How about this?

I will add a check whether the `ever failed` task information is not exist in execution summary in  the test case I have already added.